docs/user_guide/assessment.rst

```diff
@@ -468,6 +470,21 @@ argument. Here is an example of a pie chart.
 There are many other customizations that can be done. More information can be found in
 :meth:`pandas.DataFrame.plot`.
 
+
+.. _objectSavingLoading:
+
+Object Saving & Loading
+-----------------------
+
+Metrics can be saved to and loaded from files using the 'pickle <https://docs.python.org/3/library/pickle.html>' package.
+
+.. doctest:: assessment_metrics
+
+	>>> mf = MetricFrame(...)
```

Comment:
I'm afraid you need to create it with all args since this code is executed by the doc build. The doc build instructions I shared with you can be used for doing that if you exchange the html arg for doctest. Or you just run it line by line in a Python repl.
You can just define a set of small arrays for y_true, y_pred, and sensitive_features, and use a simple metric like sklearn's accuracy_score.